### DISCUSSION TITLE
Social Integration Icon too large

### DISCUSSION THREAD

**Discussion by wali-reheman** [alshedivat/al-folio#2386] [Q&A] [answered]
### Have you checked that your feature request isn't already filed?

- [X] I read through [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md) and searched through the [past issues](https://github.com/alshedivat/al-folio/issues), none of which addressed my feature request.
- [X] Yes, I have checked that this feature request isn't already filed.

### Description & Motivation

- Need a straightforward way of adjusting icon size

**Comment by george-gca** [comment, marked as answer]
Duplicate of #1100.
